src/components/forms/text-area/TextArea.vue

```diff
@@ -0,0 +1,617 @@
+
+    &:after {
+      content: '';
+      @apply absolute bottom-0 left-0 block w-full scale-x-0 border-b-2 mb-[-1px] transition-transform duration-300 border-black;
+    }
+  }
+
+  .icon {
+    @apply text-black/[0.54];
+  }
+
+  .prepend,
+  .append {
+    @apply mt-4 mx-3;
+  }
+
+  @each $color in c.$context-colors {
+    &.#{$color} {
+      .textarea {
+        &:focus {
+          + .label {
+            @apply text-rui-#{$color};
+          }
+        }
+      }
+
+      .label {
+        @apply after:border-rui-#{$color};
+      }
+
+      &.outlined {
+        .textarea {
+          &:focus {
+            ~ .fieldset {
+              @apply border-rui-#{$color};
+            }
+          }
+        }
+      }
+    }
+
+    &.text_#{$color},
+    &.with-#{$color} {
+      .prepend,
+      .append,
+      .textarea {
+        @apply text-rui-#{$color};
+      }
+
+      &:not(.disabled) .prepend svg,
+      &:not(.disabled) .append svg {
+        @apply text-rui-#{$color};
+      }
+    }
+  }
+
+  @each $color in 'error', 'success' {
+    &.with-#{$color} {
+      .textarea {
+        @apply border-rui-#{$color} #{!important};
+      }
+
+      .label {
+        @apply text-rui-#{$color} after:border-rui-#{$color} #{!important};
+      }
+
+      .fieldset {
+        @apply border-rui-#{$color} #{!important};
+      }
+    }
+  }
+
+  &.dense {
+    .textarea {
+      @apply py-1;
+      &_sizer {
+        @apply py-1;
+      }
+    }
+
+    .label {
+      @apply leading-[3.5];
+    }
+  }
+
+  &.filled,
+  &.outlined {
+    .textarea {
+      @apply px-3;
+    }
+
+    .prepend {
+      @apply mr-0;
+
+      + .inner_wrapper .textarea {
+        &:not(:placeholder-shown),
+        &:autofill,
+        &:-webkit-autofill,
+        &[data-has-value='true'],
+        &:focus {
+          + .label {
+            --x-padding: 0.75rem;
+          }
+        }
+      }
+    }
+
+    .append {
+      @apply ml-0;
+    }
+
+    .prepend,
+    .append {
+      + .inner_wrapper > .label {
+        --x-padding: 0rem;
+      }
+    }
+
+    .label {
+      --x-padding: 0.75rem;
+    }
+  }
+
+  &.filled {
+    .label {
+      @apply rounded-t bg-black/[0.06];
+    }
+
+    &.dense {
+      .textarea {
+        @apply py-2;
+        &_sizer {
+          @apply py-2;
+        }
+      }
+
+      .label {
+        @apply leading-[2.5];
+      }
+    }
+
+    &.no-label {
+      .textarea {
+        @apply py-4;
+        &_sizer {
+          @apply py-4;
+        }
+      }
+
+      &.dense {
+        .textarea {
+          @apply py-3;
+        }
+      }
+    }
+  }
+
+  &.outlined {
+    .textarea {
+      &:focus {
+        ~ .fieldset {
+          @apply border-2 border-black;
+        }
+      }
+
+      &:not(:placeholder-shown),
+      &:autofill,
+      &:-webkit-autofill,
+      &[data-has-value='true'],
+      &:focus {
+        @apply border-t-transparent;
+
+        + .label {
+          @apply leading-[1] pl-4;
```

Comment:
Is there any reason you use `leading-[1]` instead follow the one from text-field (`leading-[0]`)

This is the result when you use `leading-[1]`, I see that the result is better with `leading-0`
<img width="273" alt="image" src="https://github.com/rotki/ui-library-compat/assets/26648140/9d385769-5913-4742-82f0-8c512398f5f5">